components/elm/cime_config/config_compsets.xml

```diff
@@ -29,6 +29,11 @@
     <lname>2000_DATM%QIA_ELM%SP_SICE_SOCN_MOSART_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>IELMTEST</alias>
+    <lname>2000_DATM%QIA_ELM%SP_SICE_SOCN_SROF_SGLC_SWAV_TEST</lname>
```

Comment:
We should change `_ELM%SP_`  to `_ELM%SPBC_` because that is what is used in WCYCL compsets.

Reply:
Thanks, done.